### PR TITLE
Added chengshu's results

### PR DESCRIPTION
**Pull request by ChengshuLi**
- Final validation loss: 3.894
- Link: https://api.wandb.ai/links/chengshu/e78x7qnz
- Description: vanilla model + weight tying of input and output embeddings + hyperparameter optimization

### CODE REVIEW

**Comment by nelson-liu** [comment]
thanks!
